src/app/pages/sharing/iscsi/extent/extent-form/extent-form.component.ts

```diff
@@ -292,7 +294,9 @@ export class ExtentFormComponent implements FormConfiguration {
     this.fieldConfig = entityForm.fieldConfig;
     const extentDiskField = _.find(this.fieldConfig, { name: 'disk' }) as FormSelectConfig;
     // get device options
+    this.loader.open(this.translate.instant('Loading devices. Please wait.'));
     this.iscsiService.getExtentDevices().pipe(untilDestroyed(this)).subscribe((res) => {
+      this.loader.close();
```

Comment:
I noticed that loader is hidden if API call succeeds. However, it will stay open if API call fails. I suggest adding an failure handler as a 2nd argument for `.subscribe`, for example:

```
(res) => {
        this.loader.close();
        new EntityUtils().handleWsError(this.entityForm, res);
      },
```

Reply:
Good catch